sources/src/cache-base.ts

```diff
@@ -192,6 +192,10 @@ export class GradleStateCache {
 
         // Copy the default toolchain definitions to `~/.m2/toolchains.xml`
         this.registerToolchains()
+
+        if (core.isDebug()) {
+            this.configureInfoLogLevel()
+        }
```

Comment:
what do you think?
```suggestion
        if (core.isDebug() && !core.getInput("some-opt-out-flag")) {
            this.configureInfoLogLevel()
        }
```

Reply:
Well, I was thinking about it too but I really don't see a use-case when you want to debug your failed job but don't want see the debug logs from Gradle. You could easily skip the job output in the GitHub UI.
The whole idea is to get more logs by not changing your workflow file but simply using the debug button from the GitHub UI.
To set this input, you need to change your workflow file, and if you want this,  you can opt-out by setting the level explicitly, because this action adds the `--info` property before your custom properties.

Reply:
Nice, that sounds reasonable, could this be documented somewhere?

Use case: `--info` outputs prohibitively too many lines of logs, and GHA "stops working" with that amount.